chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/InheritanceErrorsReportEmailJob.java

```diff
@@ -169,4 +166,22 @@ private String createHtmlEmailBody(final int numRecords, final String noContentM
         }
         return htmlMessage;
     }
+
+    private String getAcbNamesAsCommaSeparatedList(JobExecutionContext jobContext) {
+        if (Objects.nonNull(jobContext.getMergedJobDataMap().getString("acb"))) {
+            return Arrays.asList(jobContext.getMergedJobDataMap().getString("acb").split("\u263A")).stream()
```

Comment:
I see `SPLIT_CHAR` defined as a constant in PendingChangeRequestEmailJob. Would it make sense to move `\u263A` to be a constant as well?

Reply:
I think it is also defined as DATA_DELIMITER in SchedulerManager